crates/viewer/re_dataframe_ui/src/display_record_batch.rs

```diff
@@ -230,17 +235,29 @@ impl DisplayComponentColumn {
                 }
             }
 
-            ctx.component_ui_registry().component_ui_raw(
-                ctx,
-                ui,
-                UiLayout::List,
-                latest_at_query,
-                ctx.recording(),
-                &self.entity_path,
-                &self.component_descr,
-                row_id,
-                data_to_display.as_ref(),
-            );
+            if let Some(variant_name) = self.variant_name {
+                ctx.component_ui_registry().variant_ui_raw(
```

Comment:
looking at the other pr the variant ui raw method does fallbacks to non-variant driven ui. Given that, could we just pass in the option to variant_ui_raw?